.github/workflows/ci.yml

```diff
@@ -1,6 +1,7 @@
 name: CI
 on:
   push:
+    branches: [ main, master ]
```

Comment:
the reason we run this on any push is because it allows to get early indication if there is a failure in the docker build. This could be due to changes in the Docker build process, or due to changes outside of our control like pip dependency problems / OS packages etc..

Reply:
right, solved! thanks